src/app/features/creature/creature-template/creature-template.component.html

```diff
@@ -535,5 +535,20 @@
         </div>
       </div> -->
     </form>
+
+    <div class="preview-container" [ngClass]="showItemPreview ? 'show-preview' : 'hide-preview'">
+      <perfect-scrollbar class="transparent">
+        <div class="creature-preview">
+          <keira-model-3d-viewer
+            [viewerType]="NPC_VIEWER_TYPE"
+            [modelId]="editorService.form.controls.modelid1.value"
+          ></keira-model-3d-viewer>
+        </div>
+      </perfect-scrollbar>
```

Comment:
if it's possible, I would avoid  using `<perfect-scrollbar>`  because it's deprecated and we should remove it completely from Keira3